socketio/src/client/client.rs

```diff
@@ -322,6 +338,84 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn socket_io_reconnect_url_auth_integration() -> Result<()> {
+        fn get_url() -> Url {
+            let timestamp = SystemTime::now()
+                .duration_since(UNIX_EPOCH)
+                .unwrap()
+                .as_millis();
+            let mut url = crate::test::socket_io_restart_url_auth_server();
+            url.set_query(Some(&format!("timestamp={timestamp}")));
+            url
+        }
+
+        let connect_num = Arc::new(AtomicUsize::new(0));
```

Comment:
Please make them static, you should be able to drop the Arc then.

Reply:
Again, is this really important when this is just for a test? This uses `Arc`'s like the existing test in the file.

Reply:
I think yes. If you treat your test code badly you will run into trouble sooner or later. Also "someone else made the same mistake" is a bad argument for code IMO.

Reply:
Fair enough.

I've changed the code and agree that looks better :)